.mergify.yml

```diff
@@ -3,6 +3,9 @@ pull_request_rules:
     conditions:
       - "status-success=ci/circleci: build"
       - "status-success=ci/circleci: noauth"
+      - "status-success=ci/circleci: install-minikube"
+      - "status-success=ci/circleci: integration-old-kind"
+      - "status-success=ci/circleci: integration-presubmit-kind"
```

Comment:
@howardjohn is this test stable?  your PR failed on it.

Reply:
It was stable, but something change in istio/istio causing the failure, I am looking into it now.